lib/tpm2_eventlog_yaml.c

```diff
@@ -128,7 +128,7 @@ bool yaml_digest2(TCG_DIGEST2 const *digest, size_t size) {
 
     return true;
 }
-bool yaml_uefi_var_unicodename(UEFI_VARIABLE_DATA *data) {
+char* yaml_uefi_var_unicodename(UEFI_VARIABLE_DATA *data) {
```

Comment:
```suggestion
char *yaml_uefi_var_unicodename(UEFI_VARIABLE_DATA *data) {
```
asterisk attached to the literal.